app/lib/forms/have_ofsted_urn.rb

```diff
@@ -0,0 +1,37 @@
+module Forms
+  class HaveOfstedUrn < Base
+    attr_accessor :has_ofsted_urn
+
+    validates :has_ofsted_urn, presence: true, inclusion: { in: %w[yes help no] }
```

Comment:
should `help` be here? or is the option missing in `next_step`?

Reply:
No it shouldn't, this was an old option in the designs that I removed everywhere except here.